### PR TITLE
Added author image for Piston article and made corrections to grammatical errors in the CompSoc article

### PR DESCRIPTION
**Pull request by suraj44**
This Pull Request contains the following changes:
1. An author image has been added for the Piston article of the 20th November week.
2. Grammatical errors in the CompSoc article of the 20th November week have been addressed.

### CODE REVIEW

File: blog_src/_posts/2017-11-20-machine-learning-for-stock-market-investment.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Here's some code on preprocessing the dataset from Quandl that would transform t
 	    ticker_list = []
      	
 	    for each_dir in stock_list[1:]:
-     		each_file = os.listdir(each_dir)
+     		each_file = os.listdir(each_dir)http://scikit-learn.org/stable/
```
Comment (salman-bhai) [inline blog_src/_posts/2017-11-20-machine-learning-for-stock-market-investment.md]: Is this correct?

**Comment by suraj44** [comment]
Resolved the issue

**Comment by salman-bhai** [comment]
PR Description ! :P

**Comment by suraj44** [comment]
I've added a description

**Pull request merged**